DataFormats/FEDRawData/src/FEDNumbering.cc

```diff
@@ -10,11 +10,11 @@ using namespace std;
 namespace {
 
   constexpr std::array<bool, FEDNumbering::FEDNumbering::MAXFEDID + 1> initIn() {
-    std::array<bool, FEDNumbering::MAXFEDID + 1> in = {{false}};
-
+    std::array<bool, FEDNumbering::MAXFEDID + 1> in = {};
     int i = 0;
-    for (i = 0; i < FEDNumbering::lastFEDId(); i++)
+    for (i = 0; i <= FEDNumbering::MAXFEDID; i++) {
```

Comment:
`FEDNumbering::lastFEDId()` is equal to `FEDNumbering::MAXFEDID`:
https://github.com/cms-sw/cmssw/blob/038eb56268f2fb41e45e46828e556c174dc1f589/DataFormats/FEDRawData/interface/FEDNumbering.h#L19